Doc/library/tkinter.ttk.rst

```diff
@@ -692,6 +692,92 @@ Bugs
 * This widget supports only "southeast" resizing.
 
 
+Spinbox
```

Comment:
I would add the description of Spinbox right after the description of Combobox. And the same for the Spinbox class and the test class.